Describe motivation in README (#192)
bmish/check-dependency-version-consistency: README.md

This plugin checks to ensure that dependencies are on consistent versions across a monorepo / yarn workspace. For example, every package in a workspace that has a dependency on `eslint` should specify the same version for it.

> Motivation:

Reviewer: ```suggestion
## Motivation:
```

> ## Motivation:

Reviewer: ```suggestion
## Motivation
```

This might still fail CI linting due to the colon.

> * **Supports a uniform developer experience.** Like shared code formatting standards, consistent dependency versions make it easier for developers to jump around a monorepo.

Reviewer: ```suggestion
* **Supports a uniform developer experience.** Like shared code formatting standards, consistent dependency versions make it easier for developers to work among different workspaces in a monorepo.
```

This plugin checks to ensure that dependencies are on consistent versions across a monorepo / yarn workspace. For example, every package in a workspace that has a dependency on `eslint` should specify the same version for it.

## Motivation

* **Supports a uniform developer experience.** Like shared code formatting standards, consistent dependency versions make it easier for developers to jump around and work across packages in a monorepo.
* **Discourages piecemeal upgrades.** When possible, a dependency should be upgraded everywhere it exists in a monorepo at the same time, and upgrades are simpler when fewer versions are present.

> * **Reduces unexpected behavior.** Having multiple, older versions present can lead to confusion, i.e. a bug present in some parts of a monorepo but not others.

Reviewer: ```suggestion
* **Reduces unexpected behavior.** Having multiple, older versions present can lead to confusion, e.g., a bug may be present in some parts of a monorepo but not others.
```

Author: This package deals only with a single workspace at a time. A workspace may have many packages inside it.

> * **Reduces unexpected behavior.** Having multiple, older versions present can lead to confusion, i.e. a bug present in some parts of a monorepo but not others, or uncertainty about which version wins out when served to the browser.

Reviewer: ```suggestion
* **Reduces unexpected behavior.** Having multiple, older versions present can lead to confusion, e.g., a bug present in some parts of a monorepo but not others, or uncertainty about which version wins out when served to the browser.
```
